test/puppetlabs/puppetdb/cli/import_export_roundtrip_test.clj

```diff
@@ -196,14 +185,3 @@
   (check-invalid-url-handling
    #(#'import/main "--host" "local:host" "--infile" "/dev/null" "--port" 10000)
    #"^Invalid destination .*"))
-
-(deftest changing-queue-destination-name
```

Comment:
Just binding a different destination name is no good. There are threads listening to the queue that need to have the correct value and HTTP requests coming in (also on a different thread) that need to get this value. A global with-redefs works better in this case, so I removed the dynamic tag and removed the test.